extensions/wikia/TriviaQuizzes/TriviaQuizzesHooks.class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class TriviaQuizzesHooks {
+    public static function onBeforePageDisplay( OutputPage $out ) {
```

Comment:
Minor: All PHP code should be using tabs, not spaces.

Reply:
Fixed in d02b0da8788